style.css

```diff
@@ -40,11 +40,16 @@ a {
   text-decoration: none;
 }
 
+.hamburger {
+  display: none;
+}
+
 .header {
-  padding-left: 260px;
-  padding-right: 260px;
-  padding-top: 359px;
+  padding-left: 300px;
+  padding-right: 300px;
```

Comment:
- [ ] [OPTIONAL] As a core requirement for this project, please use `flexbox` to align the contents of this section. 

![image](https://user-images.githubusercontent.com/84156714/185486995-a5e57ea0-ca76-4ed9-8c53-2219fc347d3b.png)